contentcuration/contentcuration/frontend/shared/views/HelpTooltip.vue

```diff
@@ -24,19 +34,15 @@
         type: String,
         default: 'info',
       },
-      small: {
-        type: Boolean,
-        default: true,
-      },
     },
   };
 
 </script>
 
 <style lang="less" scoped>
 
-  .v-icon {
-    cursor: pointer;
+  /deep/.k-tooltip {
```

Comment:
https://github.com/learningequality/kolibri-design-system/issues/595 is resolved and the KDS version with this update will be available for Studio soon. From what you said before, I think this could be removed after using the new `maxWidth` prop, is that right? Or was there something else that motivated this change?

Reply:
Yes, this can be removed once the changes introduced in this PR are available for use in Studio

Reply:
Alright, I think this PR will still be open when I create a new release tomorrow. Would using it right away in the scope of this PR work for you? Alternatively it'd be fine to have a follow-up issue - in that case I'd just ask you to explain a bit more in the code comment so whoever will be resolving has some context. Let me know what makes more sense to you.

Reply:
I guess making the changes myself might be a little more quicker and doable than explaining the entire aspect of it in a different issue/PR. I'll be happy to do the related changes in this PR itself once the new release is created tomorrow.

Reply:
Hi @BabyElias, here's the new KDS version with `maxWidth` prop on `KTooltip` you needed https://github.com/learningequality/kolibri-design-system/releases/tag/v3.2.0. You can just update package.json `kolibri-design-system` to `3.2.0` and run `yarn install`. Let me know if it's all good.

Reply:
Works perfectly! Added the prop wherever needed.